Studio-Lovelies/GG-JointJustice-Unity
# Removed redundant resolution pick menu

## Summary
Removed the redundant resolution pick menu from the main-menu's scene settings.

## Testing instructions
Open the Main Menu scene > run the game > Open settings > Check if there is only one resolution drop-down menu.

## Additional information
- `[X]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes [#470](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/issues/470)
